ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py

```diff
@@ -4,3 +4,14 @@
 
 def cat(tensors, dim=0, *, out=None):
     return ivy.concat(tensors, dim, out=out)
+
+
+def permute(input, dims, *, out=None):
```

Comment:
Sorry for my mistake before, I just found that torch.permute doesn't accept an out argument, so there need not be one in the tests